tests/test_usecases.py

```diff
@@ -469,6 +892,24 @@ def test_disable_dataset_with_db(db, schema_data, dataset_data):
         {"dataset_id": 1, "schema_id": 1, "version": "1.0"},
         update_actions_on_dataset_disablement,
     ),
+    (
+        {
+            "code": "code",
+            "version": "1.0",
+            "schema_id": 1,
+            "action": "REMOVE SCHEMA TAG",
+        },
+        update_actions_on_schema_tagging,
+    ),
+    (
+        {
+            "dataset_id": 1,
+            "schema_id": 1,
+            "version": "1.0",
+            "action": "ADD DATASET TAG",
+        },
+        update_actions_on_dataset_tagging,
+    ),
```

Comment:
Why are you keep adding cases to this? I think it is clear that since you are passing the function to test, it's not a unit test. If you think there is too much duplication, maybe all these functions should be the same function.